test/types/objectSearchCreate.test.tsx

```diff
@@ -217,4 +218,29 @@ describe('objectSearchCreate', () => {
 
     expect(onSave).toHaveBeenCalledWith({ law_firm: result });
   });
+
+  it('Properly caches and displays options', async () => {
+    const { field, searchTerm, result, props, endpoint } = getDefaults({})
+      , testerForm = await getTester(props) as any
```

Comment:
Instantiating tester to get props is a little roundabout. I would create mock props from a function; that way, when we write tests for just `ObjectSearch`, we'll have mocked props on the ready too